lib/issue_title.rb

```diff
@@ -4,12 +4,22 @@ class IssueTitle < SimpleDelegator
     "Blocker" => :blocker
   }
 
+  ESCAPED_CHARACTERS = {
+    "<" => "&lt;",
+    ">" => "&gt;",
+    "&" => "&amp;"
+  }.freeze
+
   def tags
     raw_tags.map do |tag|
       VALID_TAGS[tag]
     end.compact
   end
 
+  def to_s
+    self.gsub(/[#{ESCAPED_CHARACTERS.keys.join}]/, ESCAPED_CHARACTERS)
```

Comment:
Could we replace this with

```suggestion
    CGI.escapeHTML(self)
```

Reply:
:+1: Escaping more than required should be harmless.

Reply:
I was already using `CGI.escape`, however, according to the Slack API docu: 

> There are three characters you must convert to HTML entities and only three: &, <, and. Don't HTML entity-encode the entire message.

So I think in this case, escaping more than required is not harmless 🤔

Reply:
👍